pyproject.toml

```diff
@@ -1,10 +1,11 @@
 [tool.poetry]
 name = "ShadowFinder"
-version = "0.3.1"
+version = "0.4.0"
 description = "Find possible locations of shadows."
 authors = ["Bellingcat"]
 license = "MIT License"
 readme = "README.md"
+packages = [{ include = "shadowfinder", from = "src" }]
```

Comment:
PEP standardized way to specify the location of your package when using the `src/` layout.